neo4j/internal/collections/set.go

```diff
@@ -73,3 +73,12 @@ func (set Set[T]) Copy() Set[T] {
 	}
 	return result
 }
+
+func (set Set[T]) Contains(value T) bool {
+	for _, a := range set.Values() {
+		if a == value {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
```suggestion
func (set Set[T]) Contains(value T) bool {
	_, ok := set[value]
	return ok
}
```

please make use of the "Hash" in `HashMap` for more fast :runner::dash: